src/config/mod.rs

```diff
@@ -107,6 +107,7 @@ create_config! {
     blank_lines_lower_bound: usize, 0, false,
         "Minimum number of blank lines which must be put between items";
     edition: Edition, Edition::Edition2015, false, "The edition of the parser (RFC 2052)";
+    normalize_doc_attributes: bool, false, false, "Normalize doc attributes as doc comments";
```

Comment:
Could you move this option to line 52 please (to be next to the other comment options).